Makefile

```diff
@@ -14,72 +25,44 @@ AAR_FILE_DIR =  $(ROOT_DIR)/code/$(PROJECT_NAME)/build/outputs/aar
 init:
 	git config core.hooksPath .githooks
 
+clean:
```

Comment:
Are all the command in this makefile running?

Reply:
Seems like the make checkformat have error due to spotlessCheck.  
You can run 'spotlessApply' in Android studio grade verification.  Then it should fix that.
Others commands look good.